src/main/java/org/rsa/command/commands/ConfigureCommand.java

```diff
@@ -0,0 +1,34 @@
+package org.rsa.command.commands;
+
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.rsa.command.CommandObject;
+import org.rsa.command.SubcommandObject;
+import org.rsa.command.subcommands.configure.ReactionsSubcommand;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class ConfigureCommand extends CommandObject {
+    private static final SubcommandObject[] subcommands = { new ReactionsSubcommand() };
+
+    public ConfigureCommand()
+    {
+        super("configure", "Configures bot settings for this server.", List.of(), List.of(subcommands));
+        setIsGuildOnly();
+    }
+
+    @Override
+    public void handleSlashCommand(@NotNull SlashCommandInteractionEvent event)
+    {
+        @Nullable String subcommandName = event.getSubcommandName();
+
+        if (subcommandName != null)
+        {
+            Optional<SubcommandObject> optionalSubcommand = Arrays.stream(subcommands).filter(s -> s.getName().equals(subcommandName)).findFirst();
+            optionalSubcommand.ifPresent(subcommandObject -> subcommandObject.handleSubcommand(event));
+        }
+    }
+}
```

Comment:
Was this tested? Discord API states the commands containing subcommands will be unusable. If that is the case, this event might not need to be implemented here.

_If_ this event does trigger, should we handle a response here?

Reply:
This was tested and does work. The subcommand object deals with responding to the event.

I feel like handling the response here would just be extra work because then you'd need to type all possible return types from the subcommand (embeds, content, files, etc.) if I'm understanding you correctly. However, we could provide a default response if the event hasn't been responded to after calling the subcommand object.